### PR TITLE
feat: About Us Page

### PR DESCRIPTION
**Pull request by chinathaip**
- This is my first frontend-react work, would really appreciate suggestions and comments
- One possible further improvement is to have the content (club activities and image slideshow) dynamic and manage it through admin panel
    - maybe someone can do this later on

### CODE REVIEW

**Comment by anascence** [comment]
hello Mr.Khing i am about to review your work xD. 

First of all, the image slider looks really good. Will it be better if you add a little spacing after the slider, cus every time i try to look at it, it kinda too close to my windows taskbar. Apart from that it looks really good. 

The other thing is the "About Us" heading and "Stamford center are kinda too close or they seem off to my eyes. Maybe we don't need "About Us" header? since we are already in about us page. 

apart from that it looks cool sir. @chinathaip

File: public/assets/images/about/syntax-orientation2.png
Comment (anascence) [inline public/assets/images/about/syntax-orientation2.png:1]: Nice pic xD haha

**Review by anascence — CHANGES_REQUESTED** [review]

**Comment by Lxkas** [comment]
LGTM! Let's see what the AI has to say.
@CodiumAI-Agent /review

**Review by Lxkas — APPROVED** [review]
Great!

**Review by anascence — APPROVED** [review]

**Comment by chinathaip** [comment]
> hello Mr.Khing i am about to review your work xD.
> 
> First of all, the image slider looks really good. Will it be better if you add a little spacing after the slider, cus every time i try to look at it, it kinda too close to my windows taskbar. Apart from that it looks really good.
> 
> The other thing is the "About Us" heading and "Stamford center are kinda too close or they seem off to my eyes. Maybe we don't need "About Us" header? since we are already in about us page.
> 
> apart from that it looks cool sir. @chinathaip

Agreed, adding more space and remove the 'about us' header

**Pull request merged**